tests/backend_test.py

```diff
@@ -406,37 +430,34 @@ def test_event_rrule_this_and_future_allday_prior():
 
 
 def test_event_rrule_multi_this_and_future_allday():
-    dbi = backend.SQLiteDb('home', ':memory:', locale=LOCALE_BERLIN)
+    dbi = backend.SQLiteDb(calname, ':memory:', locale=LOCALE_BERLIN)
     dbi.update(event_rrule_multi_this_and_future_allday,
                href='event_rrule_multi_this_and_future_allday.ics', etag='abcd')
     assert dbi.list() == [('event_rrule_multi_this_and_future_allday.ics', 'abcd')]
-    events = get_allday_range(dbi, start=date(2014, 4, 30), end=date(2014, 9, 26))
-    # we are getting two events twice, because they last for two days
-    assert len(events) == 8
+    events = sorted(dbi.get_floating([calname],
+                                     datetime(2014, 4, 30, 0, 0),
+                                     datetime(2014, 9, 27, 0, 0)))
+    assert len(events) == 6
 
     assert events[0].start == date(2014, 6, 30)
     assert events[1].start == date(2014, 7, 12)
-    assert events[2].start == date(2014, 7, 12)
-    assert events[3].start == date(2014, 7, 17)
-    assert events[4].start == date(2014, 7, 19)
-    assert events[5].start == date(2014, 7, 19)
-    assert events[6].start == date(2014, 7, 24)
-    assert events[7].start == date(2014, 7, 31)
+    assert events[2].start == date(2014, 7, 17)
+    assert events[3].start == date(2014, 7, 19)
+    assert events[4].start == date(2014, 7, 24)
+    assert events[5].start == date(2014, 7, 31)
 
     assert events[0].end == date(2014, 6, 30)
     assert events[1].end == date(2014, 7, 13)
-    assert events[2].end == date(2014, 7, 13)
-    assert events[3].end == date(2014, 7, 17)
-    assert events[4].end == date(2014, 7, 20)
-    assert events[5].end == date(2014, 7, 20)
-    assert events[6].end == date(2014, 7, 24)
-    assert events[7].end == date(2014, 7, 31)
+    assert events[2].end == date(2014, 7, 17)
+    assert events[3].end == date(2014, 7, 20)
+    assert events[4].end == date(2014, 7, 24)
+    assert events[5].end == date(2014, 7, 31)
 
-    assert str(events[0].summary) == 'Arbeit'
-    for event in events[1:3] + events[4:6]:
-        assert str(event.summary) == 'Arbeit (lang)'
-    for event in events[3:4] + events[6:]:
-        assert str(event.summary) == 'Arbeit (neu)'
+    assert str(events[0].summary) == u'Arbeit'
```

Comment:
This is probably due to being based on an older `master`, but `u'` should no longer be present.